documentation/testing/test-plan.md

```diff
@@ -7,67 +7,82 @@
 |---------|---------|-----------|------------|---------------|
 | XX      | QA Name | Test Lead | 01/01/2021 | First Release |
 
+## Project Overview
+### Purpose
+The purpose of the software is to seamlessly integrate the functionalities of View My Financial Insights (VMFI) and School Financial Benchmarking (SFB). The end goal is to provide users with a unified platform where they can compare school financial data, view school spending, and analyse expenditures in comparison with similar schools. Another objective is to build data pipelines that will read data from raw data files and pass them to pipeline to store them into the schema which will later be used in the system.
 
-#  Introduction
+### Key Features and Functionalities
+The software will empower users with the ability to conduct benchmarking by comparing school financial data. Additionally, users can easily view their own school's spending patterns and compare these expenditures with similar schools or academies. There would also be a option to see a forecast of spending. Implementation of data pipelines to ingest data in the pipeline.
 
-//TODO: Brief introduction of the application under test and the development methodology.
+## Project Scope
+### In-Scope Elements for Testing
+Testing will encompass various elements, including accessibility, cross-browser functionality, automated and manual functional testing, regression testing, unit testing, integration testing, and API testing.
 
-###   Definitions and Abbreviations
+### Out-of-Scope Elements for Testing
+The performance of the system under load conditions is currently out of scope for this testing phase.
 
-//TODO: List of abbreviations and also definitions used in the document.
+## Special Attention
+A dedicated focus will be placed on testing the data upload feature to ensure that all data is accurately processed and integrated into the system.
```

Comment:
could also add in something like "ensure conformance with the required input schema"? Basically, we want to make sure that any variability in the input data due to some other user changing it is dealt with to our best capabilities to prevent any friction in the ETL process.

Reply:
done.